contracts/MultiSigWallet.sol

```diff
@@ -27,11 +27,12 @@ contract MultiSigWallet {
      *  Storage
      */
     mapping (uint => Transaction) public transactions;
+    mapping (bytes32 => uint) public swapTransactions;
```

Comment:
why do we need this map? maybe we just include bytes32 variable to the Transaction?

Reply:
Done.